src/System Application/App/Extension Management/src/ExtensionMarketplace.Codeunit.al

```diff
@@ -487,12 +487,6 @@ codeunit 2501 "Extension Marketplace"
     [EventSubscriber(ObjectType::Codeunit, Codeunit::"System Action Triggers", OpenAppSourceMarket, '', false, false)]
     local procedure OpenAppSourceMarket()
     begin
-#if not CLEAN24        
-#pragma warning disable AL0432
-        Page.Run(Page::"Extension Marketplace");
-#pragma warning restore AL0432
-#else        
-        Hyperlink('https://aka.ms/BCAppSource');
-#endif        
+        Page.Run(2515);
```

Comment:
Why hardcode with number? 
Much more readable with Page::"AppSource Product List"

Reply:
This is because Extension Management is a dependency of AppSource Apps module. If we were to do this explicit call then we would need to add Appsource Apps as a dependency of Extension Management, leading to a circular dependency. This is a way to bypass that :)

Reply:
True and agree. Missed that, of course not add circular dependency only for that :) 

It's just that you don't see which Page it is with hovering and Ctrl+Click also doesn't open it up, maybe adding a comment saying Page "xxx" in Module "xxx" for quick sight, without opening symbols and checking there. Just an idea.